backends/advanced/webui/src/components/plugins/FormField.tsx

```
@@ -0,0 +1,229 @@
+            {schema.help_text && (
+              <p className="mt-1 text-xs text-gray-500 dark:text-gray-400">
+                {schema.help_text}
+              </p>
+            )}
+          </div>
+        )
+
+      case 'password':
+        const displayValue = isMaskedValue && !isEditing ? value : value || ''
+
+        return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)